modules/ossm-applying-certificates-to-multi-cluster-topology.adoc

```diff
@@ -0,0 +1,118 @@
+// This module is used in the following assemblies:
+
+// * install/ossm-multi-cluster-topology.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-applying-certificates-to-multi-cluster-topology_{context}"]
+= Applying certificates to a multi-cluster topology 
+
+Apply root and intermediate certificate authority (CA) certificates to the clusters in a multi-cluster topology.
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. 
+====
+
+.Prerequisites
+
+* You have access to two {ocp-product-title} clusters with external load balancer support. 
+
+* You have created the root CA certificate and intermediate CA certificates for each cluster or someone has made them available for you. 
+
+.Procedure
+
+. Apply the certificates to the East cluster of the multi-cluster topology.
+
+.. Log in to East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc login -u https://<east_cluster_api_server_url>
+----
+
+.. Set up the environment variable that contains the `oc` command context for the East cluster by running the following command:
++
+[source,terminal]
+----
+$ export CTX_CLUSTER1=$(oc config current-context)
+----
+
+.. Create a project called `istio-system` by running the following command:
++
+[source,terminal]
+----
+$ oc get project istio-system --context "${CTX_CLUSTER1}" || oc  new-project istio-system --context "${CTX_CLUSTER1}"
+----
+
+.. Configure {istio} to use `network1` as the default network for the pods on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+----
+
+.. Create the CA certificates, certificate chain, and the private key for {istio} on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get secret -n istio-system --context "${CTX_CLUSTER1}" cacerts || oc create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+  --from-file=east/ca-cert.pem \
+  --from-file=east/ca-key.pem \
+  --from-file=east/root-cert.pem \
+  --from-file=east/cert-chain.pem
+----
++
+[NOTE]
+====
+If you followed the instructions in _Creating certificates for a multi-cluster mesh_ your certificates will reside in the `east/` directory. If your certificates reside in a different directory, modify the syntax accordingly. 
```

Comment:
Quotations for section references ([repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations)), and a comma:
```suggestion
If you followed the instructions in "Creating certificates for a multi-cluster mesh", your certificates will reside in the `east/` directory. If your certificates reside in a different directory, modify the syntax accordingly. 
```

Reply:
done